backend/src/api/model/block/mutations.rs

```diff
@@ -379,6 +411,12 @@ pub(crate) struct UpdateSeriesBlock {
     order: Option<VideoListOrder>,
 }
 
+#[derive(GraphQLInputObject)]
+pub(crate) struct UpdateVideoBlock {
+    title: Nullable<String>,
+    event: Option<Id>,
```

Comment:
Why is `event` optional again?

Reply:
You could also use this mutation to just update the title. Or other potential future fields. Every top level field in these `Update*Block` structures should be optional.